### PR TITLE
First round of implementation

### PR DESCRIPTION
**Pull request by mvs5465**
Includes:
- Kafka listener
- Rest depositor
- Configurability via application.properties or environment variables
- Travis CI (https://travis-ci.org/usdot-jpo-ode/jpo-sdw-depositor)
- SonarCloud analysis (https://sonarcloud.io/dashboard?id=usdot.jpo.ode%3Ajpo-sdw-depositor%3Adev)

Todo items that require more information from SDW: (EDIT: complete)
- [x] SDW schema
- [x] Implement username/password in schema

### CODE REVIEW

**Comment by mvs5465** [comment]
Closing to continue development, will recreate later.

File: src/main/java/jpo/sdw/depositor/consumerdepositors/KafkaConsumerRestDepositor.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+         for (ConsumerRecord<String, String> record : records) {
+
+            long offset = record.offset();
+            String key = record.key();
+            String value = record.value();
+            String destination = this.getRestDepositor().getDestination().toString();
+
+            JSONObject jsonMsg = new JSONObject();
+            jsonMsg.put("systemDepositName", "string");
+            jsonMsg.put("encodeType", "hex");
```
Comment (hmusavi) [inline src/main/java/jpo/sdw/depositor/consumerdepositors/KafkaConsumerRestDepositor.java]: `encodeType` could be `hex` or `base64`. Need to be able to support both.

File: src/main/java/jpo/sdw/depositor/consumerdepositors/KafkaConsumerRestDepositor.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            jsonMsg.put("encodedMsg", value);
+
+            logger.info("Depositing message to {} KafkaOffset = {}, KafkaKey = {}, MessageValue = {}", destination,
```
Comment (hmusavi) [inline src/main/java/jpo/sdw/depositor/consumerdepositors/KafkaConsumerRestDepositor.java]: This should be a `debug` log.

File: src/main/java/jpo/sdw/depositor/depositors/SDWDepositor.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      super(restTemplate, destination);
+   }
+
+   @Override
+   public void deposit(String message) {
+
+      try {
+         ResponseEntity<String> result = this.getRestTemplate().postForEntity(this.getDestination(), message,
+               String.class);
+         logger.info("Response received. Status: {}, Body: {}", result.getStatusCode(), result.getBody());
```
Comment (hmusavi) [inline src/main/java/jpo/sdw/depositor/depositors/SDWDepositor.java]: Should probably be a `debug` log too.

File: src/main/resources/application.properties [hunk 4]
```diff
@@ -0,0 +1,16 @@
+#General Properties
+#==================
+#groupId=${project.groupId}
+groupId=usdot.jpo.ode
```
Comment (hmusavi) [inline src/main/resources/application.properties]: hmmm. Somehow there is `^M` character here. Please delete.

**Pull request merged**